docs/dev/cli/saucectl/jobs/get.md

```diff
@@ -0,0 +1,70 @@
+---
+id: get
+title: saucectl jobs get
+sidebar_label: saucectl jobs get
+---
+
+## Description
+
+Get job details from Sauce Labs.
+
+## Usage
+
+<span className="cli">$ saucectl jobs get jobID [flags]</span>
```

Comment:
I know this is consistent with the format we're using for the cli docs, but I think `jobID` should be visually distinct from the command, otherwise it looks like its a verbatim part of the command.

`$ saucectl jobs get <jobID> [flags]`
or
`$ saucectl jobs get JOB_ID [flags]`

First is the format in git docs, second is the format in docker docs.

Just my two cents.

Reply:
Good point! I'll use the second one. `<jobID>` would be treated as an HTML element here.